source/fundamentals/indexes.txt

```diff
@@ -144,13 +150,44 @@ created above.
    :language: js
    :start-after: begin-query
    :end-before: end-query
+   :dedent:
 
 Multikey indexes behave differently from non-multikey indexes in terms of
 query coverage, index bound computation, and sort behavior. For a full
 explanation of multikey indexes, including a discussion of their behavior
 and limitations, refer to the :manual:`Multikey Indexes page
 </core/index-multikey>` in the MongoDB manual.
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify the ``clusteredIndex`` option in
+the ``CollectionOption``. The ``clusteredIndex`` option must specify the
+``_id`` field as the key, and the unique field as ``true``.
+
+The following example uses the ``collection()`` method to create a
```

Comment:
Issue: The example does not use the collection() method
Suggestion: Change to "uses the createCollection() method"